heroku_connect/db/models/base.py

```diff
@@ -4,7 +4,27 @@
 from . import fields
 from ...conf import settings
 
-__all__ = ('HerokuConnectModel',)
+__all__ = ('HerokuConnectModel', 'registry')
+
+
+class HerokuConnectModelRegistry:
```

Comment:
We already have a `utils.get_heroku_connect_models` wich uses Django's buildin model registry. No need to add yet another one ;)

Reply:
I thought it would be nicer to not always iterate over every connected model and transform its `db_table` to match a `table_name` from the trigger log. But I guess you might call the registry a premature optimisation; it definitely shares some of the functionality of `get_heroku_connect_models`.